infra/bridge/src/utils/webhook-helpers.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+
+const { GrowthCampaign } = require('@origin/growth-campaign/src/models')
```

Comment:
I'm just not sure the growth-campaign package will get used anywhere else than here so wondering if worth splitting it from growth package...
So as opposed to creating a growth-campaign package, have you evaluated the alternative of querying growth.originprotocol.com to get the campaign data ? It seems it could be simpler... ?

Reply:
Yep. I didn't think about querying. That seems like far better solution. Will change it.